charts/cofide-connect/values.schema.json

```diff
@@ -143,6 +143,49 @@
             }
           ]
         },
+	"datastore": {
```

Comment:
`dataStore` ought to have `required = ["cloudSQL"]`, similarly the nested `cloudSQL` and `oidc` ought to have the `required` set.

Since the default values sets them but with empty values this doesn't impose any additional burden on consumers of the chart - but means in our usage of the values file we don't have to worry about checking for property existence.